packages/eas-cli/src/commands/release/create.ts

```diff
@@ -0,0 +1,97 @@
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import log from '../../log';
+import { findProjectRootAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+import { getProjectIdAsync } from '../../submissions/commons';
+
+type UpdateRelease = {
+  id: string;
+  releaseName: string;
+};
+
+async function createUpdateReleaseOnAppAsync({
+  appId,
+  releaseName,
+}: {
+  appId: string;
+  releaseName: string;
+}): Promise<UpdateRelease> {
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .mutation<
+        { updateRelease: { createUpdateReleaseForApp: UpdateRelease } },
+        { appId: string; releaseName: string }
+      >(
+        gql`
+          mutation CreateUpdateReleaseForApp($appId: ID!, $releaseName: String!) {
+            updateRelease {
+              createUpdateReleaseForApp(appId: $appId, releaseName: $releaseName) {
+                id
+                releaseName
+              }
+            }
+          }
+        `,
+        {
+          appId,
+          releaseName,
+        }
+      )
+      .toPromise()
+  );
+  return data.updateRelease.createUpdateReleaseForApp;
+}
+
+export default class ReleaseCreate extends Command {
+  static description = 'Create a release on the current project.';
+
+  static args = [
```

Comment:
Could you also implement the `--json` flag to get only JSON output?

Reply:
What would the output look like in this case?

```
{
  accountName: string;
  projectName: string;
  releaseName: string;
}
```
?

Reply:
I think prefixing property names with the type is a code smell - if we want all these we should nest the objects.

But I think we should maybe just return the release object or some subset of its fields. Especially since this is a CRUD method for release. The only thing you actually need if you're using this for scripting is the `id` (and maybe `name` if we generate the default name somehow). So it could be something like:
```ts
{
  id: string,
  name: string,
}
```